src/CLI/renderer/templates/show_vlan.j2

```diff
@@ -0,0 +1,23 @@
+{% set vars = {'vlanName': ""} %}
+{% set vars = {'ifName': ""} %}
+{% set vars = {'tagMode': ""} %}
+{% set vars = {'tagLetter': ""} %}
+{% if json_output -%}
+Q: A - Access (Untagged), T - Tagged
+{{'%-11s'|format("NUM")}}{{'%-2s'|format("Q Ports")}}
+{% for vlanKey, memberEntries in json_output.items() %}
+{% if vars.update({'vlanName':vlanKey}) %}{% endif %}
+{% for ifKey, ifMode in memberEntries.items() %}
+{% if vars.update({'ifName':ifKey}) %}{% endif %}
+{% if vars.update({'tagMode':ifMode}) %}{% endif %}
+{% if ifMode == "untagged" %}
+{% if vars.update({'tagLetter':" U "}) %}{% endif %}
+{% else %}
+{% if vars.update({'tagLetter':" T "}) %}{% endif %}
+{% endif %}
+{{'%-10s'|format(vars.vlanName)}}{{'%-2s'|format(vars.tagLetter)}}{{'%-10s'|format(vars.ifName)}}
```

Comment:
We may have to use short form of interface (e.g Eth0 instead of Ethernet0 but this conflicts with "mgmt intf"), we should have the logic to combine multiple intfs in a range and show here.

Reply:
Yes, I have considered that behavior similar to what we have in OS10. I will handle it later.

Reply:
Please make sure we follow the below format for "show vlan" output, for example, instead of Vlan100, use just 100 under Num column.

S6K-12# show vlan
Codes: * - Default VLAN, M - Management VLAN, R - Remote Port Mirroring VLANs,
       @ ? Attached to Virtual Network
Q: A - Access (Untagged), T - Tagged
    NUM    Status    Description                     Q Ports
*   1      Active                                    A Eth1/1/2-1/1/3,1/1/6-1/1/8,1/1/10-1/1/32
                                                     A Po10
    10     Inactive
    20     Inactive
    100    Inactive
    200    Inactive                                  T Eth1/1/10
S6K-12#

Reply:
Yes, will take care of it. Earlier, we didn't had oper-status support.
127.0.0.1:6379> hgetall VLAN_TABLE:Vlan101
1) "admin_status"
2) "up"
3) "mtu"
4) "9100"
127.0.0.1:6379> 
This is how, it was earlier.
I think broadcom added the fix for providing oper-status info as well, so I can add that in HLD and final CLI.